src/seedu/addressbook/AddressBook.java

```diff
@@ -940,13 +1006,14 @@ private static String encodePersonToString(String[] person) {
      * @param personData person string representation
      * @return whether format of add command arguments allows parsing into individual arguments
      */
+    //Changed indexes to constants for better understanding
```

Comment:
This kind of temporary comments should be entered as comments in GitHub, not as java comments.